apps/lldb/dot.lldbinit.in

```diff
@@ -0,0 +1,3 @@
+# Put this file in your home directory as .lldbinit
```

Comment:
Given that this file also allows comment, please add an ASF header as in the py file

Reply:
Done